eng/tox/create_package_and_install.py

```diff
@@ -165,21 +173,39 @@ def build_and_discover_package(setuppy_path, dist_dir, target_setup, package_typ
         default="wheel",
     )
 
+    parser.add_argument(
+        "--pre-download-disable",
+        dest="pre_download",
+        help="During a dev build, we will restore package dependencies from a dev feed before installing them. The presence of this flag disables that behavior.",
+        action="store_true",
+    )
 
     args = parser.parse_args()
 
+    commands_options = []
+    built_pkg_path = ""
     setup_py_path = os.path.join(args.target_setup, "setup.py")
+    additional_downloaded_reqs = []
+    tmp_dl_folder = os.path.join(args.distribution_directory, "dl")
+    os.mkdir(tmp_dl_folder)
+
+    # If extra index URL is passed then set it as argument to pip command
+    if args.extra_index_url:
+        commands_options.extend(["--extra-index-url", args.extra_index_url])
+
+    # preview version is enabled when installing dev build so pip will install dev build version from devpos feed
+    if args.install_preview or os.getenv("SetDevVersion"):
```

Comment:
Can we consistently use either SetDevVersion or --pre-download-disable to trigger this fork?

Reply:
Will definitely give it a shot.